### PR TITLE
Remove the Meta/Content courier + Include Template Helper

### PR DESCRIPTION
**Pull request by bordoni**
### 🎫 Ticket

[ECP-1695]

### 🎥 Artifacts 
https://i.bordoni.me/9fKSBt8s

[ECP-1695]: https://stellarwp.atlassian.net/browse/ECP-1695?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Events/Integrations/Plugins/Elementor/Template/Controller.php [hunk 1]
```diff
@@ -81,71 +91,34 @@ public function add_filters(): void {
...
-		// Prevents the clone of the Base template from happening all the time.
-		if ( ! empty( $value ) ) {
-			return $value;
-		}
-
-		if ( ! is_string( $key ) ) {
-			return $key;
-		}
+	public function include_template_selection_helper(): void {
+		$preview = tribe_get_request_var( 'elementor-preview' );
```
Comment (Camwyn) [inline src/Events/Integrations/Plugins/Elementor/Template/Controller.php:104]: Reasons for using this vs `Plugin::$instance->preview->is_preview_mode()`? 
More reliable?

File: src/admin-views/integrations/plugins/elementor/template-selection-helper.php [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+ * This template is used to inject a helper message to the Elementor editor to help users understand that they are editing,
+ * the JavaScript and CSS are inline to avoid having the loading complexity for Elementor Edit area.
+ *
+ * @since TBD
+ *
+ * @package TEC\Events\Integrations\Plugins\Elementor\Template
+ */
+
+?>
+<style>
```
Comment (Camwyn) [inline src/admin-views/integrations/plugins/elementor/template-selection-helper.php:14]: Why add styles? 
For an injected element they could as easily be inline.
The styles are on the page even if the element doesn't get injected.

**Review by Camwyn — APPROVED** [review]
Couple notes, but acceptable.

File: src/admin-views/integrations/plugins/elementor/template-selection-helper.php (hunk 2 above)
Comment (bordoni) [reply src/admin-views/integrations/plugins/elementor/template-selection-helper.php:14]: True I will do that.

File: src/Events/Integrations/Plugins/Elementor/Template/Controller.php (hunk 1 above)
Comment (bordoni) [reply src/Events/Integrations/Plugins/Elementor/Template/Controller.php:104]: Yeah more reliable.

File: src/admin-views/integrations/plugins/elementor/template-selection-helper.php (hunk 2 above)
Comment (bordoni) [reply src/admin-views/integrations/plugins/elementor/template-selection-helper.php:14]: Ahh it makes the JS funky. Lets keep it that way for now.

**Pull request merged**